index.html

```diff
@@ -68,75 +65,78 @@
 					</a>
 				</div>
 			</div>
+
+		</nav>
+		
+		<div class="tildeContainer">
+			<div class="tildeSpacer">
+				<div>
+				</div>
+			</div>
 			
-			<div class="navSpacer">
+			<div class="tildeBox">
 				<div>
-					
+					~
```

Comment:
Use the html entity code `&tilde;` in your html to avoid parsing errors (I haven't seen any, but it's good practice!)